datamol/fp.py

```diff
@@ -20,25 +21,28 @@
 from rdkit.DataStructs.cDataStructs import ExplicitBitVect
 from rdkit.DataStructs.cDataStructs import ULongSparseIntVect
 
+_FP_GENERATORS = {
+    "ecfp": rdFingerprintGenerator.GetMorganGenerator,
+    "fcfp": rdFingerprintGenerator.GetMorganGenerator,
+    "topological": rdFingerprintGenerator.GetTopologicalTorsionGenerator,
+    "atompair": rdFingerprintGenerator.GetAtomPairGenerator,
+    "rdkit": rdFingerprintGenerator.GetRDKitFPGenerator,
+    "ecfp-count": rdFingerprintGenerator.GetMorganGenerator,
+    "fcfp-count": rdFingerprintGenerator.GetMorganGenerator,
```

Comment:
some fp have disappeared here. Are their deprecated ?

Reply:
They are still [there](https://github.com/datamol-io/datamol/blob/18ddc51c57214441971bd81c1da2696d353bb328/datamol/fp.py#L37). I put fingerprint generators altogether, because they are called in different manner in `to_fp` function.